### ISSUE TITLE
CSS not loaded?

### ISSUE THREAD

**Issue by ghost** [trustworktech/create-react-ssr-app#5]
How can i fix this?

node v: 10.15.3
npm: 6.4.1

![image](https://user-images.githubusercontent.com/48561012/55927593-2f270d80-5c48-11e9-90f9-9b2b5a45a4cb.png)

![image](https://user-images.githubusercontent.com/48561012/55927622-4a921880-5c48-11e9-9c58-4b8a9fefa59f.png)

![image](https://user-images.githubusercontent.com/48561012/55928554-b5911e80-5c4b-11e9-8b67-bd4904719383.png)

**Comment by maxparelius** [comment]
@vncnzo is this happening after you first create the project? What version of react-ssr-scripts are you using?

**Comment by ghost** [comment]
Yes, first time and I try to fix it but I don't know how.

![image](https://user-images.githubusercontent.com/48561012/56012514-6ddbc700-5d1f-11e9-9475-eca925edbc08.png)

**Comment by maxparelius** [comment]
@vncnzo what commands did you run and steps to repro?

**Comment by ghost** [comment]
npx create-react-ssr-app, npm start. Idk why it doesnt include css and import png, svg etc. I uninstall node, reinstall to try if it will be fixed but nothing changed.

**Comment by maxparelius** [comment]
@vncnzo its a strange error looks like just cannot connect to the webpack dev server. Are there any errors on the terminal?

**Comment by ghost** [comment]
Just the DeprecationWarning. Tapable plugin, use new Api on hooks

**Comment by maxparelius** [comment]
@vncnzo are you able to run `create-react-app`? Try same commands with create react app.

```
npx create-react-app my-app
cd my-app
npm start
```

**Comment by ghost** [comment]
yes, no errors.

**Comment by maxparelius** [comment]
@vncnzo can you open the network tab and take screen shot of the requests going out for the css resource.

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/48561012/56156619-335c7d80-5ff0-11e9-87d2-0ec0814b2cc6.png)


![image](https://user-images.githubusercontent.com/48561012/56156851-a9f97b00-5ff0-11e9-8615-d73aa00f4a9a.png)

**Comment by maxparelius** [comment]
@vncnzo whats the response of that request

**Comment by ghost** [comment]
nothing

![image](https://user-images.githubusercontent.com/48561012/56157064-2c823a80-5ff1-11e9-8930-a05adac1b7b0.png)

**Comment by maxparelius** [comment]
@vncnzo can you screen shot the same request for create-react-app?

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/48561012/56158380-686acf00-5ff4-11e9-8540-a5766b2c9e11.png)

**Comment by maxparelius** [comment]
@vncnzo Ok thanks I think I know what is happening will get a fix out soon and let you know.

**Comment by ghost** [comment]
Thank you bro!

**Comment by maxparelius** [comment]
@vncnzo can you upgrade `react-ssr-scripts` and try again? `npm install --save react-ssr-scripts`

**Comment by ghost** [comment]
It works now. But when I refresh/load it has a delay to render the styles, is it normal?

![image](https://user-images.githubusercontent.com/48561012/56186363-beba2b00-6051-11e9-92e2-b4981ec6c2c1.png)

**Comment by maxparelius** [comment]
@vncnzo yea its just in dev mode with the way the styles are loaded. Not an issue in production.

**Comment by patryk-bernasiewicz** [comment]
Have you guys by any chance found a workaround or a fix for that? I've stumbled upon the exact same problem, neither the JS bundle will load for me or any of the CSS files.


![image](https://user-images.githubusercontent.com/10709214/64884095-23110300-d661-11e9-8482-5dd35dc86c04.png)

I've created the project using `create-react-ssr-app` freshly installed this evening, I'm using it with TypeScript. Tried to re-install the react-ssr-scripts package, but nothing changed.

When I open the project for the first time after running the `yarn start` command, it sometimes displays an error message saying "Cannot parse app.html" or something like that (cannot reproduce now, but will post a screenshot when it happens again).

**EDIT**
There it is, the "parsing app.html" problem:
![image](https://user-images.githubusercontent.com/10709214/64884585-5ef89800-d662-11e9-8d9f-b92e8385d52e.png)
